src/view/transaction-detail.js

```diff
@@ -46,6 +46,11 @@ const TransactionDetailView = ({ store, nav }) => (
         <DetailField name="Status">
           {store.selectedTransaction.statusLabel}
         </DetailField>
+        {store.selectedTransaction.preimage ? (
```

Comment:
Should we add this to the mobile transaction view as well?

Reply:
@valentinewallace good catch! I added it in 6bd1900